pages/Documents/Reporting/OutboundConfigurationAPI/overview.md

```diff
@@ -0,0 +1,400 @@
+
+
+### Outbound Configuration API
+
+API for account level outbound configurations
+
+**1. Account - Get outbound configuration  for the given account**
+
+Click [**Account**](https://proactive-messaging.z1.fs.liveperson.com/api/api-docs/?api=reporting#/Account/get) to go through API spec to get started.
+
+| Method | URI                                                                   |
+| :----- | :-------------------------------------------------------------------- |
+| GET    | https://{domain}/api/account/{id}/app/{appName}/handoffs/list?v=1 |
+
+**Path Parameters**
+
+| Name      | Description        | Required | Value/Example                                                                    |
+| :-------- | :----------------- | :------- | :------------------------------------------------------------------------------- |
+| domain    | domain             | Yes      | va.handoff.liveperson.ne or lo.handoff.liveperson.ne or sy.handoff.liveperson.ne |
+| accountId | LivePerson site ID | Yes      | 12345678                                                                         |
+| app       | App name           | Yes      | "prmsg"                                                                          |
+| v         | Version            | Yes      | 1                                                                                |
+
+**Query Parameters**
+
+| Name   | Description                                   | Required | Value/Example |
+| :----- | :-------------------------------------------- | :------- | :------------ |
+| limit  | Max count of outbound configurations to fetch | No       | 30            |
+| offset | Starting point to fetch from                  | No       | 0             |
+
+**Request Headers**
+
+| Header        | Description                                                                                                                                                                                                                                         | Value/Example    |
+| :------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :--------------- |
+| Content-Type  | Used to indicate the media type of the resource                                                                                                                                                                                                     | application/json |
+| Authorization | [OAuth 2.0](https://developers.liveperson.com/connector-api-send-api-authorization-and-authentication.html#get-appjwt) or [OAuth 1.0](https://developers.liveperson.com/retrieve-api-keys-create-a-new-api-key.html) (Section 8) or LE Bearer token |                  |
+
+**Response Example**
+
+200 Success
```

Comment:
Please restructure these examples:

1. Do without HTML line breaks.

2. Do without lists.

Example:

```
#### Response example

##### 200 Success [case okay here, because HTTP reference]

SMS Twilio:

```json
…
```

…
```